OrbitCore/Path.cpp

```diff
@@ -255,12 +255,6 @@ std::string Path::GetHome() {
   std::string home = GetEnvVar("HOME") + "/";
   return home;
 }
-
```

Comment:
nit: maybe explain in the commit message that it is only being used on the service side anyways.

Reply:
will do on rebase